### ISSUE TITLE
Waypoint flexibility in 'Edit Waypoints' window and Secondary Targets for AI

### ISSUE THREAD

**Issue by nosv1** [dcs-liberation/dcs_liberation#1307]
### Ability to rearrange waypoint order
Something like dragging waypoints before or after each other in the waypoint editor. It's often I'll organize a DEAD flight for multiple targets, or something similar, but to edit the order of the waypoints I'd have to recreate the flight and re-add the waypoints. 

#### Add non-target waypoints
If this is possible, and while we're editing waypoint stuff, maybe a way to add an 'ingress, nav, and egress' waypoints, and then when you decide what these 'nontarget waypoints' go between it auto-generates its location - of course, the user can then edit the location in the map view.

#### Set Waypoint Altitudes
And if we're still whacking out waypoint features, the ability to set the altitude for waypoints (prob just an up/down arrow right there in that cell for that waypoint). At some point, it feels like we're just recreating a better ME :D

### Ability to add secondary targets for AI
Like, you could say, "go take out this boat, then when you're done, come over and help out with the front line."
The first idea that comes to mind is a button by the 'recreate as xyz' that says maybe 'add additional xyz' and then boom, another target added - obvi it's not that simple behind the scenes, but that's the idea.

I have no idea how AI handles multiple targets if you were to do it in ME, therefore, I have no idea how possible this is in the Liberation generation.

**Comment by HILOK** [comment]
yes, changing FL/angels in increments of 1000ft using up/down arrows would be a great addition
